src/BloomExe/Book/BookStarter.cs

```diff
@@ -419,14 +417,10 @@ public static void SetupIdAndLineage(XmlElement parentPageDiv, XmlElement childP
 			}
 		}
 
-
-
-
-
 		private string GetInitialName(string sourcePath, string parentCollectionPath)
 		{
 			var storage = _bookStorageFactory(sourcePath);
-			var name = storage.Dom.GetMetaValue("defaultNameForDerivedBooks",  Path.GetFileName(sourcePath));
+			var name = storage.Dom.GetMetaValue("defaultNameForDerivedBooks",  "Book");
```

Comment:
Thanks for catching that. It still does have a use, it turns out; e.g. if you make a new book from "Vaccinations", the file and folder will be named Vaccinations until you enter a translated title. But this nicety is so small that I've now removed it entirely, and you instead get a ui-localized translation of the word "Book". We can then experiment with it and then strip out the commented stuff when we have some confidence.